.goreleaser.yml

```diff
@@ -0,0 +1,80 @@
+# This is an example goreleaser.yaml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+env:
+  - CGO_ENABLED=0
+before:
+  hooks:
+    # We strongly recommend running tests to catch any regression before release.
+    # Even though, this an optional step.
+    - go test ./...
+    # As part of the release doc files are included as a separate deliverable for
+    # consumption by Packer.io. To include a separate docs.zip uncomment the following command.
+    #- make ci-release-docs
+    # Check plugin compatibility with required version of the Packer SDK
+    # - make plugin-check
```

Comment:
The scaffolding repo (https://github.com/hashicorp/packer-plugin-scaffolding) originally had this `make plugin-check` enabled and required a GNUmakefile from the same repo. I tried that myself and had a few errors, so I've bypassed it for now.